src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx

```diff
@@ -63,6 +73,9 @@ export class VerifyCollectorStep extends PureComponent<Props> {
             text="Finish"
             type={ButtonType.Submit}
             testID="next"
+            onClick={() => {
+              this.props.history.push(`/orgs/${orgID}/load-data/telegrafs`)
+            }}
```

Comment:
let's pull this method out into an instance method since it doesn't rely on any data computed while rendering and it's inefficient to recreate identical anonymous functions every render.

Reply:
Ah right I keep forgetting about this! I'll do that 👍